### PR TITLE
Site development

### LINKED ISSUE #13: Add RELEASING.md
Add a release instruction document ala https://github.com/dblock/waffle/blob/master/Docs/MavenReleaseGuide.md.

### PR DESCRIPTION
**Pull request by claudiow**
Sorry for being AWOL, some work plus a long week end.

I did not manage to have AppVeyor working for SNAOSHOT and Releases.

I agree we should release ASAP, I created a manual release document see [this](https://github.com/claudiow/log4jna/blob/site-development/MavenRelease.md)

If you follow the instructions you should be able to release from your local machine after the PR

This closes #13 and #6 but nothing else. The site stills needs work but is in good shape for publishing.

I will continue with the site, #37 and AppVeyor, but at a slower pace as I have a tight deadline project on my hands.

### CODE REVIEW

File: APPVEYOR.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+cache:
+  # Cache Maven and m.2
+  - '%MAVEN_HOME%'
+  - '%M2%'
+
+...
+``` 
+
+ ## <a name="b1"></a>Log4JNA Default Build
```
Comment (dblock) [inline APPVEYOR.md]: Doesn't seem to render as markdown. Probably because of the whole `<a name` business.

File: APPVEYOR.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+ This is a very straight forward build, we build whenever te code or other files changes in the `master` branch.
```
Comment (dblock) [inline APPVEYOR.md]: straightforward is spelled as one word, type in `the` code

**Comment by dblock** [comment]
I'll do a spelling/English pass on this and merge.

File: ENVIRONMENT.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+#### Eclipse
+See [this article](https://books.sonatype.com/m2eclipse-book/reference/creating-sect-importing-projects.html) to import the Maven projects.
+
+#### IteliJ IDEA
+See [this article](https://www.jetbrains.com/help/idea/2016.1/importing-project-from-maven-model.html) to import the Maven projects.
+
+#### NetBeans
+See [this article](http://wiki.netbeans.org/MavenBestPractices) to import the Maven projects.
+
+### Set up the Registry manually
```
Comment (dblock) [inline ENVIRONMENT.md]: Is this still needed? Tests pass without this AFAIK.

**Comment by dblock** [comment]
Merged via https://github.com/dblock/log4jna/commit/0c263099db0058151f2f5adca9ff1be59451cba8.

File: ENVIRONMENT.md (hunk 3 above)
Comment (claudiow) [inline ENVIRONMENT.md]: This are just instructions on how to set up the development environment. 

I think that we should leave references to setting up the registry manually, it does not harm and might avoid questions from users.

**Comment by claudiow** [comment]
Are you performing the release?

**Comment by dblock** [comment]
@claudiow I am not. It's all yours!

**Comment by claudiow** [comment]
Do I have access to your repo?

The tag and gh-pages should go there?

Will my ssh key work on your repo?

**Comment by dblock** [comment]
Yes you do. You should be able to create a branch and push to it.

**Pull request closed without merging**